scripts/repl.js

```diff
@@ -4,11 +4,11 @@
 const enchannelZMQ = require('../src');
 const createShellSubject = enchannelZMQ.createShellSubject;
 const createIOPubSubject = enchannelZMQ.createIOPubSubject;
-const uuid = require('uuid');
+const uuid = require('uuid/v4');
 const path = require('path');
 
 const chalk = require('chalk');
-const identity = uuid.v4();
+const identity = uuid();
```

Comment:
The only thing I'm not too pleased with in this PR is the inconsistency of using `uuid()` vs. `v4()`. I'm leaning towards `const uuidv4 = require('uuid/v4');`